conceptual.md

```diff
@@ -0,0 +1,46 @@
+## Conceptual Exercise
+
+Answer the following questions below:
+
+#### What are some ways of managing asynchronous code in JavaScript?
+* Using Async Functions (Async / Await)
+* Promises
+* Callback functions
+
+#### What is a Promise?
+* an object that may produce a single value sometime in the future.  Can return a resolved value or error/reason its not resolved
+
+#### What are the differences between an async function and a regular function?
+* The only difference between the two functions is that asyncFn is declared with the async keyword. This means that Javascript will make sure that the asnycFn will return with a Promise (either resolved or rejected) even if an error occured in it  

```

Comment:
On a lower level, regular funcs block the event loop, but async funcs don't until you await them.